app/views/games/new.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Join a Game</h1>
+<p>Click to join a game with available seats.</p>
+
+<%= simple_form_for @game, :url; => games_path do |f| %>
+    <%= f.input :black_player_id, :collection => User.all %>
```

Comment:
`User.all` is dangerous because if we have a lot of users (imagine thousands), it loads them all into memory.

For this I'd do a hidden field and set it to `current_user.id`